### ISSUE TITLE
deneme

### ISSUE THREAD

**Issue by okandalan** [okandalan/Tic-Tac-Toe#1]
`x = 2 + 5`

**Comment by okandalan** [comment]
Hello,

Thank you for your work. When I try to give more than one numerical feature as input, I get this error:
split_with_sizes expects split_sizes to sum exactly to 4 (input tensor's size at dimension -1), but got split_sizes=[1, 1] 
It is related to this line "base, index = torch.split(input_x, [1, 1], dim=-1)" on the forward method of the FLEembedding class.

There is another issue related to the same problem for SASRecF but it was closed. Thus, I opened new one

My yaml file:
`#Environment Settings
seed: 2020
reproducibility: True
data_path: "dataset/"
dataset: RentTheRunway
show_progress: True
model: DIN

#Data settings
USER_ID_FIELD: user_id
ITEM_ID_FIELD: item_id
TIME_FIELD: timestamp

load_col:
  inter: [user_id, item_id, rating, timestamp, fit, rented_for]
  user: [user_id, age, bust_size, weight, height, body_type]
  item: [item_id, category, size]

MAX_ITEM_LIST_LENGTH: 50

numerical_features: [age, weight, size]

threshold:
  rating: 8

#Training Settings
epochs: 50
train_batch_size: 4096
learner: sgd
learning_rate: 3e-3
mlp_hidden_size: [256, 256, 256]

train_neg_sample_args:
  sample_num: none

#Evaluation Settings
eval_args:
  split: { "LS": "valid_and_test" }
  order: TO
  group_by: user
  mode: labeled

metrics: ["AUC", "LogLoss"]

eval_batch_size: 131072
`

**Comment by okandalan** [comment]
```
asdasda
asdasd
asdads
```

**Comment by okandalan** [comment]
`#Environment Settings
seed: 2020
reproducibility: True
data_path: "dataset/"
dataset: RentTheRunway
show_progress: True
model: DIN

#Data settings
USER_ID_FIELD: user_id
ITEM_ID_FIELD: item_id
TIME_FIELD: timestamp

load_col:
  inter: [user_id, item_id, rating, timestamp, fit, rented_for]
  user: [user_id, age, bust_size, weight, height, body_type]
  item: [item_id, category, size]

MAX_ITEM_LIST_LENGTH: 50

numerical_features: [age, weight, size]

threshold:
  rating: 8

#Training Settings
epochs: 50
train_batch_size: 4096
learner: sgd
learning_rate: 3e-3
mlp_hidden_size: [256, 256, 256]

train_neg_sample_args:
  sample_num: none

#Evaluation Settings
eval_args:
  split: { "LS": "valid_and_test" }
  order: TO
  group_by: user
  mode: labeled

metrics: ["AUC", "LogLoss"]

eval_batch_size: 131072
`

**Comment by okandalan** [comment]
```
#Environment Settings
seed: 2020
reproducibility: True
data_path: "dataset/"
dataset: RentTheRunway
show_progress: True
model: DIN

#Data settings
USER_ID_FIELD: user_id
ITEM_ID_FIELD: item_id
TIME_FIELD: timestamp

load_col:
  inter: [user_id, item_id, rating, timestamp, fit, rented_for]
  user: [user_id, age, bust_size, weight, height, body_type]
  item: [item_id, category, size]

MAX_ITEM_LIST_LENGTH: 50

numerical_features: [age, weight, size]

threshold:
  rating: 8

#Training Settings
epochs: 50
train_batch_size: 4096
learner: sgd
learning_rate: 3e-3
mlp_hidden_size: [256, 256, 256]

train_neg_sample_args:
  sample_num: none

#Evaluation Settings
eval_args:
  split: { "LS": "valid_and_test" }
  order: TO
  group_by: user
  mode: labeled

metrics: ["AUC", "LogLoss"]

eval_batch_size: 131072

```

**Comment by okandalan** [comment]
![Screenshot from 2023-12-11 14-54-42](https://github.com/okandalan/Tic-Tac-Toe/assets/70844540/79059661-959a-48a9-b5c6-e9233a2b0cd6)
